worker/utils.py

```diff
@@ -252,7 +252,7 @@ def get_code_from_tds(code_id, code=False, dynamics_only=False):
     content_object = {}
 
     for name, blocks in file_names.items():
-        name = name.split("/")[-1]
+        # name = name.split("/")[-1]
```

Comment:
I'm curious what's going on with this commented out line?

Reply:
This was causing problems in repo's with deeper file trees. It was not able to find the correct files when fetching the code with this in place, we can just remove that line.